workflows/prognostic_c48_run/tests/test_regression.py

```diff
@@ -28,6 +28,24 @@
         "name": "diags.zarr",
         "times": {"kind": "interval", "frequency": 900, "times": None},
     },
+    {
+        "name": "monitored_tendencies_and_storages.zarr",
+        "times": {"kind": "interval", "frequency": 900, "times": None},
+        "variables": [
+            "storage_of_specific_humidity_path_due_to_fv3_physics",
+            "storage_of_specific_humidity_path_due_to_python",
+            "storage_of_total_water_path_due_to_fv3_physics",
+            "storage_of_total_water_path_due_to_python",
+            "tendency_of_air_temperature_due_to_fv3_physics",
+            "tendency_of_air_temperature_due_to_python",
+            "tendency_of_eastward_wind_due_to_fv3_physics",
+            "tendency_of_eastward_wind_due_to_python",
+            "tendency_of_northward_wind_due_to_fv3_physics",
+            "tendency_of_northward_wind_due_to_python",
+            "tendency_of_specific_humidity_due_to_fv3_physics",
+            "tendency_of_specific_humidity_due_to_python",
```

Comment:
these variables also end up getting output in `diags.zarr` since that diagnostic file outputs all variables.